.vscode/settings.json

```diff
@@ -1,7 +1,19 @@
 {
-  "eslint.format.enable": true,
-  "eslint.probe": ["javascript", "typescript", "html", "markdown", "typescriptreact"],
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
I generally run the formatter through ESLint

Reply:
ESLint w/ prettier plugin I'm assuming? Makes sense, happy to switch that over (but would prefer you do it since I don't know how it was configured before and I _genuinely don't give a shit about linting lmao_)